doc/user-guide/Jingle-SIP-setup.md

```diff
@@ -0,0 +1,300 @@
+Please replace the default host defined in file `etc/ejabberd.cfg` in line
+
+```erlang
+{hosts, ["localhost"] }.
+```
+
+to:
+
+```erlang
+{hosts, ["xmpp.example", "sip.example"] }.
+```
+
+Now we need to enable `mod_jingle_sip`, please add the following line in modules list (somewhere around line 740 in the same file)
+
+```erlang
+{mod_jingle_sip, [{proxy_host, "sip.example"}]},
+```
+
+More details on MongooseIM configuration you can find in [Basic Configuration](../Basic-configuration.md)
+and in [Modules configuration](../advanced-configuration/Modules.md)
+
+Now we are registering both users in MongooseIM by calling the following commands:
+
+    bin/mongooseimctl register xmpp.user xmpp.example test_pass
+    bin/mongooseimctl register sip.user sip.example test_pass
+
+Yes, we need to have the `sip.user@sip.example` registered in MongooseIM.
+This is needed because a Jingle call can be initiated by a regular XMPP client only when the app knows the other user's full JID.
+The easiest way to achieve that is to exchange presence information between these 2 users.
+This can happen automatically if 2 xmpp users have each other in the roster.
+
+The roster can be set by us with the following commands:
+
+    bin/mongooseimctl add_rosteritem sip.user sip.example xmpp.user xmpp.example xmpp.user none both
+    bin/mongooseimctl add_rosteritem xmpp.user xmpp.example sip.user sip.example sip.user none both
+
+### Adding users to Jitsi
+
+Now the `sip.user@sip.example` has to be added to Jitsi app.
+When the app is opened for the first time it will display a window to configure the user.
+Later users can be configured from the `Preferences` page.
+
+#### Adding a SIP user
+
+In order to add a user who connects to the SIP server we need to choose the `SIP` protocol from the available `networks` in Jitsi.
+In the `SIP id` field we put `sip.user@sip.example` and in the `Password` field we put `1234` as in the `agents.yml` file.
+Now we need to switch to `Advanced` options and go to the `Connection` tab.
+Here we need to unselect the `Configure proxy automatically` and put the IP of our `Routr` server, port number `5060` and `TCP` as the preferred transport.
+
+![SIP setup in Jitsi](jingle-sip/jitsi-sip-config.png)
+
+#### Adding an XMPP user
+
+Now we have to add `sip.user@sip.example` to Jitsi's XMPP network in order to connect this user to MongooseIM over XMPP.
+It's very similar to adding a user to Jitsi's SIP network, the only difference is the password,
+for the XMPP conection it's `test_pass` as set when registering the user in MongooseIM.
+Here we also need to go to the `Advanced` window and the `Connection` tab in order to put the IP addres (the same as before) in the `Connect Server` field.
+Remember to check the `Override server default options` box.
+
+To connect `sip.user@sip.exmple` to MongooseIM over XMPP is to cheat Jingle a bit, so that the client app for user `sip.xmpp@xmpp.example` can start the Jingle call.
```

Comment:
Not 100% sure, but it should probably be "to cheat ON Jingle". @goddammit, can you correct me if I'm wrong?